src/client/src/tests/SceneForm.js

```diff
@@ -27,13 +27,33 @@ o.spec('SceneForm', function() {
     o(out.should.contain('new name')).equals(true)
     
     // make sure that change name form actually changes the name in the data model
-    out.trigger('.save-name-form', 'onsubmit', {preventDefault: () => {}}) // trigger a submit event on the form
+    out.click('.save-changes-button')
```

Comment:
Following my previous comments, instead of triggering a click on the button element, you will need to trigger the onsubmit event on the form element.